bootstrapper/internal/kubernetes/kubernetes.go

```diff
@@ -329,9 +330,30 @@ func (k *KubeWrapper) StartKubelet() error {
 		return fmt.Errorf("starting kubelet: %w", err)
 	}
 
-	k.log.Info("Prioritizing etcd I/O")
-	if err := k.etcdIOPrioritizer.PrioritizeIO(); err != nil {
-		k.log.Warn("Prioritizing etcd I/O failed", "error", err)
+	// We just started the kubelet service here, so let's give it some time to start etcd.
+	ticker := time.NewTicker(1 * time.Second)
+	defer ticker.Stop()
+	timeout, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+	for {
```

Comment:
Wouldn't it be easier to handle this wait and retry logic in the etcdIO client itself?
I also think in this case it would be best to have `PrioritizeIO` not return any errors at all, but simply log potential failures.